src/controllers/index.js

```diff
@@ -0,0 +1,6 @@
+const { getMovies, getApiMovies, postMovies } = require('./movies');
+const { handleClintError, handleServerError } = require('./error')
+
+
+
```

Comment:
make sure to use a formatter, you have a lot of empty spaces between your code blocks which is kinda not consistent, for example here one empty line would be nice
```js
const { getMovies, getApiMovies, postMovies } = require('./movies');
const { handleClintError, handleServerError } = require('./error')

module.exports = { handleClintError, handleServerError, getMovies, getApiMovies, postMovies }; 

```